<launch>
  <param name="/use_sim_time" value="false"/>

  <!-- Publishes the voxel grid to rviz for display -->
  <node pkg="costmap_2d" type="costmap_2d_markers" name="voxel_visualizer">
    <remap from="voxel_grid" to="costmap/voxel_grid"/>
  </node>

  <!-- Run the costmap node -->
  <node name="costmap_node" pkg="costmap_2d" type="costmap_2d_node" >
    <rosparam file="$(find cob_3d_experience_mapping)/tools/costmap.yaml" command="load" ns="costmap" />
  </node>

  <!--node name="local_path_planning" pkg="cob_3d_experience_mapping" type="local_path_planning" >
    <remap from="costmap" to="/costmap_node/costmap/costmap"/>
    <remap from="action" to="/robot0/cmd_vel"/>
  </node-->
  
</launch>
